server.js

```diff
@@ -17,8 +35,53 @@ app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+app.get('/', (req, res) => {
+  res.send('Welcome to my Happy Thoughts API');
```

Comment:
A great way to show the endpoint is to do the express-list-endpoints. Maybe you can check that out :) It's super user friendly!